lighthouse-core/index.js

```diff
@@ -73,5 +73,6 @@ lighthouse.getAuditList = Runner.getAuditList;
 lighthouse.traceCategories = require('./gather/driver').traceCategories;
 lighthouse.Audit = require('./audits/audit');
 lighthouse.Gatherer = require('./gather/gatherers/gatherer');
+lighthouse.NetworkRecords = require('./computed/network-records.js');
```

Comment:
Wanted to call this change out as the only real functionality change, but fairly large impact on our public API :)

Reply:
yeah at that point I think we should restructure a lot of our exports and expose some aliases so folks can `require('lighthouse/types')` `require('lighthouse/computed')` etc and we're free to move around files as we wish